### PR TITLE
rustdoc: Make some functions private that don't need to be public

### PR DESCRIPTION
**Pull request by jyn514**
r? @GuillaumeGomez

### CODE REVIEW

**Comment by jyn514** [comment]
LMK if I should remove the `debug!`, I added it while looking at https://github.com/rust-lang/rust/issues/77763.

File: src/librustdoc/clean/utils.rs [hunk 1]
```diff
@@ -109,13 +109,14 @@ pub fn get_deprecation(cx: &DocContext<'_>, def_id: DefId) -> Option<Deprecation
...
 
-pub fn external_generic_args(
+fn external_generic_args(
     cx: &DocContext<'_>,
     trait_did: Option<DefId>,
     has_self: bool,
     bindings: Vec<TypeBinding>,
     substs: SubstsRef<'_>,
 ) -> GenericArgs {
+    debug!("cleaning bindings {:?} for external trait {:?}", bindings, trait_did);
```
Comment (GuillaumeGomez) [inline src/librustdoc/clean/utils.rs:119]: Is it really necessary to add this `debug`?

Comment (jyn514) [reply src/librustdoc/clean/utils.rs]: Ok, I removed it.

**Comment by GuillaumeGomez** [comment]
Thanks!

@bors: r+ rollup

**Pull request merged**